packages/office-ui-fabric-react/src/components/Layer/Layer.base.tsx

```diff
@@ -93,71 +170,47 @@ export class LayerBase extends BaseComponent<ILayerProps, {}> {
       this._host = host;
 
       if (!this._layerElement) {
-        const rootElement = this._rootElement.current;
-        const doc = getDocument(rootElement);
-
-        if (!doc || !rootElement) {
+        const doc = getDocument();
```

Comment:
I believe getDocument takes an element, so if host is available, maybe pass that in?

Reply:
@cschleiden Do you remember the rationale here?